tests/parsing/GlobbingTest.py

```diff
@@ -100,6 +101,41 @@ def test_alternatives(self):
                              sorted(alternatives))
 
 
+class EscapingTest(unittest.TestCase):
+
+    def _test_cases(self, in_arr, out_arr):
```

Comment:
please call this `assert_cases` or similar, at least it should start with `assert` so one directly sees this is a test assertion^^